include/inviwo/qt/editor/inviwomainwindow.h

```diff
@@ -75,7 +77,7 @@ class IVW_QTEDITOR_API InviwoMainWindow : public QMainWindow, public NetworkEdit
     void showWindow();
 
     void openLastWorkspace(std::string workspace = "");
-    void openWorkspace(QString workspaceFileName);
+    bool openWorkspace(QString workspaceFileName);
```

Comment:
Returns true if?

Reply:
Might rename askOpenWorkspace?

Reply:
added comment, returns false in case the workspace could not be opened.